app/models/concerns/github_identity.rb

```diff
@@ -23,11 +23,11 @@ def hidden=(val)
 
   def github_settings_url
     if private_repo_token.present?
```

Comment:
could also simplify with `key = if private_repo_token.present?` instead of per-line